native_client/deepspeech.h

```diff
@@ -157,7 +157,32 @@ DEEPSPEECH_EXPORT
 int DS_EnableExternalScorer(ModelState* aCtx,
                             const char* aScorerPath);
 
+  /**
+ * @brief Enable decoding with hot-word boosting.
+ *
+ * @param aCtx The ModelState pointer for the model being changed.
+ * @param aHotWords A list of hot words.
```

Comment:
This is wrong: "a list of comma-separated" hot words describes what you have, but in fact it should really be a list of words, a comma-separated string here feels like a huge hack